### PR TITLE
fix: Pin yarn version

### LINKED ISSUE #279: Local build fails
## Bug description

Can't make the app `yarn start` locally

### Steps to reproduce 

1. Git clone [my fork](https://github.com/MichalBryxi/vegancheck.me) via `git clone git@github.com:MichalBryxi/vegancheck.me.git`
2. See [the installation manual](https://frontendnetwork.github.io/vegancheck.me/)
3. Run `yarn install` with success:
```
yarn install
yarn➤ YN0000: ┌ Resolution step
➤ YN0000: ⠇ resolve-from@npm:^4.0.0
➤ YN0032: │ fsevents@npm:2.3.2: Implicit dependencies on node-gyp are discouraged
➤ YN0061: │ @npmcli/move-file@npm:2.0.1 is deprecated: This functionality has been moved to @npmcli/fs
➤ YN0061: │ request@npm:2.88.2 is deprecated: request has been deprecated, see https://github.com/request/request/issues/3142
➤ YN0061: │ har-validator@npm:5.1.5 is deprecated: this library is no longer supported
➤ YN0061: │ uuid@npm:3.4.0 is deprecated: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
➤ YN0061: │ rollup-plugin-terser@npm:7.0.2 is deprecated: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-terser
➤ YN0061: │ @babel/polyfill@npm:7.12.1 is deprecated: 🚨 This package has been deprecated in favor of separate inclusion of a polyfill and regenerator-runtime (when needed). See the @babel/polyfill docs (https://babeljs.io/docs/en/babel-polyfill) for more information.
➤ YN0061: │ sourcemap-codec@npm:1.4.8 is deprecated: Please use @jridgewell/sourcemap-codec instead
➤ YN0061: │ core-js@npm:2.6.12 is deprecated: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.
➤ YN0002: │ eslint-config-sznm@npm:1.1.1 [56ae9] doesn't provide prettier (p84e6e), requested by eslint-plugin-prettier
➤ YN0002: │ next-pwa@npm:5.6.0 [56ae9] doesn't provide @babel/core (p89f09), requested by babel-loader
➤ YN0002: │ next-pwa@npm:5.6.0 [56ae9] doesn't provide webpack (p2d30c), requested by babel-loader
➤ YN0002: │ next-pwa@npm:5.6.0 [56ae9] doesn't provide webpack (pa166f), requested by clean-webpack-plugin
➤ YN0002: │ next-pwa@npm:5.6.0 [56ae9] doesn't provide webpack (p7c542), requested by terser-webpack-plugin
➤ YN0002: │ next-pwa@npm:5.6.0 [56ae9] doesn't provide webpack (p7a00b), requested by workbox-webpack-plugin
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed in 11s 110ms
➤ YN0000: ┌ Fetch step
➤ YN0013: │ workbox-strategies@npm:6.5.4 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ workbox-streams@npm:6.5.4 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ workbox-sw@npm:6.5.4 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ workbox-webpack-plugin@npm:6.5.4 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ workbox-window@npm:6.5.4 can't be found in the cache and will be fetched from the remote registry
➤ YN0066: │ typescript@patch:typescript@npm%3A5.0.4#optional!builtin<compat/typescript>::version=5.0.4&hash=1f5320: Cannot apply hunk #4
➤ YN0000: └ Completed in 15s 695ms
➤ YN0000: ┌ Link step
➤ YN0000: │ ESM support for PnP uses the experimental loader API and is therefore experimental
➤ YN0007: │ core-js@npm:2.6.12 must be built because it never has been before or the last one failed
➤ YN0000: └ Completed in 2s 575ms
➤ YN0000: Done with warnings in 29s 505ms
```
4. Run `yarn start` with following *errors*:
```
yarn start
ready - started server on 0.0.0.0:1030, url: http://localhost:1030
Error: Could not find a production build in the '/Users/michal/Project/vegancheck.me/.next' directory. Try building your app with 'next build' before starting the production server. https://nextjs.org/docs/messages/production-start-no-build-id
    at NextNodeServer.getBuildId (/Users/michal/Project/vegancheck.me/.yarn/__virtual__/next-virtual-a5a3f11195/3/.yarn/berry/cache/next-npm-13.3.1-f9884a3d2e-9.zip/node_modules/next/dist/server/next-server.js:352:23)
    at new Server (/Users/michal/Project/vegancheck.me/.yarn/__virtual__/next-virtual-a5a3f11195/3/.yarn/berry/cache/next-npm-13.3.1-f9884a3d2e-9.zip/node_modules/next/dist/server/base-server.js:146:29)
    at new NextNodeServer (/Users/michal/Project/vegancheck.me/.yarn/__virtual__/next-virtual-a5a3f11195/3/.yarn/berry/cache/next-npm-13.3.1-f9884a3d2e-9.zip/node_modules/next/dist/server/next-server.js:166:9)
    at NextServer.createServer (/Users/michal/Project/vegancheck.me/.yarn/__virtual__/next-virtual-a5a3f11195/3/.yarn/berry/cache/next-npm-13.3.1-f9884a3d2e-9.zip/node_modules/next/dist/server/next.js:167:24)
    at async /Users/michal/Project/vegancheck.me/.yarn/__virtual__/next-virtual-a5a3f11195/3/.yarn/berry/cache/next-npm-13.3.1-f9884a3d2e-9.zip/node_modules/next/dist/server/next.js:187:31
    at async NextServer.prepare (/Users/michal/Project/vegancheck.me/.yarn/__virtual__/next-virtual-a5a3f11195/3/.yarn/berry/cache/next-npm-13.3.1-f9884a3d2e-9.zip/node_modules/next/dist/server/next.js:149:24)
    at async startServer (/Users/michal/Project/vegancheck.me/.yarn/__virtual__/next-virtual-a5a3f11195/3/.yarn/berry/cache/next-npm-13.3.1-f9884a3d2e-9.zip/node_modules/next/dist/server/lib/start-server.js:269:13)
    at async nextStart (/Users/michal/Project/vegancheck.me/.yarn/__virtual__/next-virtual-a5a3f11195/3/.yarn/berry/cache/next-npm-13.3.1-f9884a3d2e-9.zip/node_modules/next/dist/cli/next-start.js:77:5)
```
5. Run `yarn build` with following *errors*:
```
yarn build
info  - Linting and checking validity of types .Failed to compile.

./src/components/Scanner/scanner.tsx:1:34
Type error: Cannot find module 'react' or its corresponding type declarations.

> 1 | import React, { Component } from "react";
    |                                  ^
  2 | import Quagga from "@ericblade/quagga2";
  3 |
  4 | interface ScannerProps {
```
6. Run `next build` with following *errors*:
```
next build
zsh: command not found: next
```

### System

```sh
node --version
v18.14.0

yarn --version
4.0.0-rc.38

volta --version
1.1.1
```

## Additional info

Hope this helps. Happy to provide more information if needed.

### PR DESCRIPTION
**Pull request by MichalBryxi**
- The build is not working with `yarn@berry`
- This change pins the version down to `v1.x`
- Assumes usage of `volta`, but also makes the fact self-documenting

Fixes #279

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,5 +30,9 @@
     "eslint": "8.39.0",
     "eslint-config-next": "^13.3.4",
     "eslint-config-sznm": "^1.1.1"
+  },
+  "volta": {
```
Comment (philipbrembeck) [inline package.json:34]: Nice!

**Review by philipbrembeck — APPROVED** [review]

**Comment by philipbrembeck** [comment]
Hi Michal, I just got to thank you again 😊 
Quick and simple fix, but it should work for now. I'll be updating the docs accordingly.

**Comment by philipbrembeck** [comment]
Unfortunately, this broke production (again). Has nothing to do with the PR in itself, it's just that we currently have an OPs problem.

Will try this later again.

**Comment by philipbrembeck** [comment]
Hi @MichalBryxi, could you please send in this PR again? Thank you very much!

**Comment by MichalBryxi** [comment]
Sure thing: #319

It's literally the same commit :) Let me know if you meant it in a different way.

**Pull request merged**